corehq/apps/reports/standard/cases/case_list_explorer.py

```diff
@@ -193,22 +193,12 @@ def send_email_to_dev_more(domain, user, query, total_results):
 
     ¯\_(ツ)_/¯
     """
-
-    message = """
-    Hi Dev! Someone just performed a query with the case list explorer. Cool!
-
-    Domain: {}
-    User: {}
-    Query: {}
-    Total Results: {}
-
-    Yours truly,
-    CLEBOT
-    """.format(domain, user, query if query else "Empty Query", total_results)
-
-    send_mail_async.delay(
-        subject="Case List Explorer Query Performed",
-        message=message,
-        from_email=settings.DEFAULT_FROM_EMAIL,
-        recipient_list=["@".join(['dmore', 'dimagi.com'])],
-    )
+    _assert = soft_assert(["@".join(['dmore', 'dimagi.com'])],
```

Comment:
@djmore9 I'm switching this "Case List Explorer" email to a `soft_assert` so that it's disabled by default on non-Dimagi environments.  It should still have all the same info, but the format will look different.  It still goes just to you, and the subject should look like:

> [production] Soft Assert: [75c324b6] Case List Explorer Query Performed